lib/simple_form/dropdown_select/version.rb

```diff
@@ -1,5 +1,5 @@
 module SimpleForm
   module DropdownSelect
-    VERSION = "0.1.4"
+    VERSION = "0.2.0"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.